engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/engine/embedded/requestresponse/RequestResponseDeploymentStrategy.scala

```diff
@@ -80,23 +79,22 @@ class RequestResponseDeploymentStrategy(httpConfig: HttpBindingConfig, config: R
 
   override def onScenarioAdded(jobData: JobData,
                                parsedResolvedScenario: CanonicalProcess)(implicit ec: ExecutionContext): Try[RequestResponseDeployment] = synchronized {
-    import pl.touk.nussknacker.engine.requestresponse.FutureBasedRequestResponseScenarioInterpreter._
-
-    val interpreter = RequestResponseInterpreter[Future](parsedResolvedScenario, jobData.processVersion, contextPreparer, modelData, Nil,
-      ProductionServiceInvocationCollector, ComponentUseCase.EngineRuntime)
-    val interpreterWithSlug = slugForScenario(jobData.metaData).product(interpreter)
-    interpreterWithSlug.foreach { case (slug, interpreter) =>
-      slugToScenarioRoute += (slug -> new ScenarioRoute(new RequestResponseAkkaHttpHandler(interpreter), config, jobData.processVersion.processName))
-      interpreter.open()
+    // RequestResponseScenarioInterpreter is 'opened' in constructor of RequestResponseRunnableScenarioInterpreter
+    lazy val interpreterTry = Try(new RequestResponseRunnableScenarioInterpreter(jobData, parsedResolvedScenario, modelData = modelData, contextPreparer = contextPreparer, requestResponseConfig = config))
```

Comment:
modeData and contextPreparer params have the same names as params in function declaration, maybe write just
new RequestResponseRunnableScenarioInterpreter(jobData, parsedResolvedScenario, modelData, contextPreparer, requestResponseConfig = config)